safe_flag is broken by only testing the preprocessor so it can't reject this...
classic makefiles
--
--- a/Makefile
+++ b/Makefile
@@ -19,7 +19,7 @@
 
 CPPFLAGS := $(CPPFLAGS) -D_GNU_SOURCE -I include
 SHARED_FLAGS := -pipe -O3 -flto -fPIC -fvisibility=hidden -fno-plt \
-    -fstack-clash-protection $(call safe_flag,-fcf-protection) -fstack-protector-strong \
+    -fstack-clash-protection -fstack-protector-strong \
     -Wall -Wextra $(call safe_flag,-Wcast-align=strict,-Wcast-align) -Wcast-qual -Wwrite-strings \
     -Wundef
 
